cmd/operator/main.go

```diff
@@ -125,6 +126,7 @@ func parseFlags(fs *flag.FlagSet) {
 
 	fs.StringVar(&kubeletObject, "kubelet-service", "", "Service/Endpoints object to write kubelets into in format \"namespace/name\"")
 	fs.Var(&kubeletSelector, "kubelet-selector", "Label selector to filter nodes.")
+	fs.Var(&nodeAddressPriority, "node-address-priority", "Node address priority. Either internal or external.")
```

Comment:
maybe prefix with `kubelet-` to make it obvious that it affects the kubelet controller config.
```suggestion
	fs.Var(&nodeAddressPriority, "kubelet-node-address-priority", "Node address priority. Either 'internal' or 'external'. Default: 'internal'.")
```